features/config/upgrade/src/test/java/org/opennms/config/upgrade/LiquibaseUpgraderIT.java

```diff
@@ -66,41 +70,70 @@
 @RunWith(OpenNMSJUnit4ClassRunner.class)
 @TestExecutionListeners({TemporaryDatabaseExecutionListener.class})
 @ContextConfiguration(locations = {
-        "classpath:/META-INF/opennms/applicationContext-minimal-conf.xml"})
+        "classpath*:/META-INF/opennms/component-dao.xml",
+        "classpath:/META-INF/opennms/applicationContext-soa.xml",
+        "classpath:/META-INF/opennms/applicationContext-minimal-conf.xml",
+        "classpath:/META-INF/opennms/applicationContext-config-service.xml"})
+
 @JUnitTemporaryDatabase
 public class LiquibaseUpgraderIT implements TemporaryDatabaseAware<TemporaryDatabase> {
 
+    private final static String SCHEMA_NAME = "provisiond";
+    private final static String CONFIG_ID = "provisiond";
```

Comment:
Although it is a test, we do need to think about how to unify the schema name and config id.

Reply:
I saw you had somewhere else:
 SCHEMA_NAME = "provisiond";
CONFIG_ID = "default";
Is that what you mean? I am good with that.